.vscode/launch.json

```diff
@@ -114,7 +114,8 @@
             "args": [
                 "runserver",
             ],
-            "django": true
+            "django": true,
+            "justMyCode": false
```

Comment:
This allows you to put breakpoints in source code on VSCode. I've found it useful when trying to figure out how stuff works.